guides/introduction/templates.md

```diff
@@ -1,16 +1,16 @@
 # Template
 
-We call template to almost every file in the input folder. These templates
+We call template to almost any file in the input folder. Templates
 are transformed, and combined, to create the website.
 
-## Helpers
+## View Helpers
```

Comment:
Should we rename this "Template Helpers" here?

Reply:
I would rather do it altogether in another change.